src/frontend-samples/issues-sample/IssuesWidget.tsx

```diff
@@ -0,0 +1,504 @@
+
+    /** Loop through the dates and put them together in chunks */
+    return combinedByDayOrdered.map((date) => (
+      // eslint-disable-next-line react/jsx-key
+      <div className="date-group">
+        <div className="date">
+          <span>{date}</span>
+        </div>
+        {combinedByDay[date]}
+      </div>
+    ));
+  };
+
+  return (
+    <>
+      <div style={{ height: "300px" }}>
+        {/** When the issues haven't loaded yet, display spinner */}
+        {issues.length === 0 &&
+          <Spinner size={SpinnerSize.Medium} />
+        }
+
+        {/** When the issues are loaded, display them in a list */}
+        {!currentIssue && issues && Object.keys(previewImages).length > 0 && issues.length > 0 &&
+          <div>
+            <Subheading style={{ margin: "0", padding: "8px 5px", color: "#fff" }}>{`Issues (${issues.length})`}</Subheading>
+            {issues.map((issue: IssueGet) => {
+              const createdDate = issue.createdDateTime ? new Date(issue.createdDateTime).toLocaleDateString() : undefined;
+              const binaryUrl = issue.displayName && previewImages[issue.displayName] ? URL.createObjectURL(previewImages[issue.displayName]) : undefined;
+              const imageStyle = binaryUrl ? { backgroundImage: `url(${binaryUrl})` } : {};
+              return (
+                // eslint-disable-next-line react/jsx-key
+                <div className="issue">
+                  <div className="issue-preview">
+                    {issue.modelView &&
+                      <div className="thumbnail" style={imageStyle} onClick={async () => applyView(issue)}>
+                        <span className="open icon icon-zoom" title={"Locate & Zoom"} />
+                      </div>
+                    }
+                    <div className="issue-status" style={{ borderTop: `14px solid ${issue.properties._StatusColor ?? "red"}`, borderLeft: `14px solid transparent` }} />
+                  </div>
+                  <div className="issue-info" onClick={() => setCurrentIssue(issue)}>
+                    <Leading className={"issue-title"}>{`${issue.number} | ${issue.subject}`}</Leading>
+                    <div className="issue-subtitle">
+                      <span className={"assignee-display-name"}>{issue.assignee?.displayName}</span>
+                      <div className={"created-date"}>
+                        <span>{createdDate}</span>
+                      </div>
+                    </div>
+                  </div>
+                </div>
+              );
+            })}
+          </div>
+        }
+
+        {/** When an issue is slected from the inital list, show the tab interface */}
+        {currentIssue &&
+          <div className={"issue-details"}>
+            <div className={"header"}>
+              <IconButton styleType='borderless' size='small' onClick={() => { setCurrentIssue(undefined); setActiveTab(0); }}><img src=".\\progress-backward_3.svg" /></IconButton>
```

Comment:
Why not use something like `<span className="icon icon-chevron-right"></span>` instead of `<img>`. That way we wont need the public asset.

Reply:
Look at collapse gallery/code panel buttons

Reply:
Fixed.